broker/src/main/java/org/apache/rocketmq/broker/processor/AdminBrokerProcessor.java

```diff
@@ -285,7 +273,19 @@ private synchronized RemotingCommand updateAndCreateTopic(ChannelHandlerContext
 
         this.brokerController.registerIncrementBrokerData(topicConfig, this.brokerController.getTopicConfigManager().getDataVersion());
 
-        return null;
+        response.setCode(ResponseCode.SUCCESS);
+        return response;
+    }
+
+    private boolean isSystemTopic(RemotingCommand response, String topic) {
+        if (this.brokerController.getTopicConfigManager().isSystemTopic(topic)) {
+            String errorMsg = "the topic[" + topic + "] is conflict with system reserved words.";
+            log.warn(errorMsg);
```

Comment:
You can combine two code into one and use placeholder like `log.warn("xxxxxxx { } xxxx", topic)`

Reply:
errorMsg used in log and response's remark, so it is better to use a local var to store it

Reply:
> errorMsg used in log and response's remark, so it is better to use a local var to store it

You are right